README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://vuestic.dev" target="_blank">
-      <img alt="Vuestic UI Logo" width="400" src="packages/ui/src/assets/vuestic-ui-logo.png">
+      <img alt="Vuestic UI Logo" width="400" src=".github/assets/vuestic-ui-logo.png">
```

Comment:
Why? Not big deal, but not exclusive to github (used in docs as well). So we might want to put in docs.

Reply:
We shouldn't store docs assets in UI package anyway. Since it is related to all packages, we can store it it .github, just because we have this folder, so we do not create new one.